content/ja/post/li-kvm-forum-21/index.md

```diff
@@ -0,0 +1,37 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+
+title: "Mr. Guoqing Li and Dr.Dario Faggioli's Collabrative Talk at KVM FORUM 2021"
+subtitle: ""
+summary: ""
+authors: ["Guoqing Li"]
+tags: ["Cloud Computing", "Virtualization Performance", "KubeVirt", "Kubernetes"]
```

Comment:
Please change `"Cloud Computing"` to `"Cloud"`. This way the post will be linked from this page:
https://sdlab.naist.jp/tags/cloud/